tools/install-powershell.sh

```diff
@@ -128,9 +130,16 @@ install(){
         else
         #Script files are not local - pull from remote
         echo "Could not find \"appimage.sh\" next to this script..."
-        echo "Pulling it from \"$gitreposcriptroot/appimage.sh\""
-        bash <(wget -qO- $gitreposcriptroot/appimage.sh) $@
-    fi
+        echo "Pulling and executing it from \"$gitreposcriptroot/appimage.sh\""
+        if [ -n "$(command -v curl)" ]; then
```

Comment:
how universal is `command -v`?

Reply:
Here is a discussion of it's portability versus other methods: https://stackoverflow.com/questions/592620/how-to-check-if-a-program-exists-from-a-bash-script

Reply:
Thanks